pkg/cli/completion_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright2020TheKubernetesAuthors.
+
+LicensedundertheApacheLicense,Version2.0(the"License");
+youmaynotusethisfileexceptincompliancewiththeLicense.
+YoumayobtainacopyoftheLicenseat
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unlessrequiredbyapplicablelaworagreedtoinwriting,software
+distributedundertheLicenseisdistributedonan"ASIS"BASIS,
+WITHOUTWARRANTIESORCONDITIONSOFANYKIND,eitherexpressorimplied.
+SeetheLicenseforthespecificlanguagegoverningpermissionsand
+limitationsundertheLicense.
+*/
+
+package cli
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("CLI", func() {
+	var (
+		c   *CLI
+		err error
+	)
+
+	BeforeEach(func() {
+		c = &CLI{}
+	})
+
+	Context("newBashCmd", func() {
```

Comment:
@yashsingh74 The tests could be written in way in which we test the various scenarios and assert if error has occurred or not, we need not add an additional check of `err != nil` before assertion.

```go
var _ = Describe("CLI test", func(){
          var (
                  // necessary variables
          )

          When("testing command completions", func() {
                  It("testing x", func() {
                       cmd = c.newBashCmd()
                       Expect(cmd.Execute()).NotTo(HaveOccured())
                   })
           })
})
```

More details on writing test cases using ginkgo can be seen here: https://onsi.github.io/ginkgo/

Reply:
`Expect(cmd.Execute()).NotTo(HaveOccured())` is an example, you could see more declarative statements like these in the doc